packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,76 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import styles from "./Tooltip.module.css";
+import { Popover, PopoverProps } from "../popover/Popover";
+import { cssColor } from "@stenajs-webui/theme";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+const variantIcons = {
+  info: stenaInfoCircle,
+  warning: stenaExclamationCircle,
+  error: stenaExclamationTriangle,
+};
+
+export interface TooltipProps
+  extends Partial<Omit<PopoverProps, "theme" | "render">> {
   label: string;
+  maxWidth?: number;
+  variant?: TooltipVariant;
+  children: JSX.Element;
 }
 
-export const Tooltip: React.FC<TooltipProps> = ({ label, ...popoverProps }) => {
+const TooltipText: React.FC<{ label: string; maxWidth?: number }> = ({
+  label,
+  maxWidth,
+}) => (
+  <Indent spacing={0.5} display={"inline-block"}>
+    <Text
+      color={cssColor("--lhds-color-ui-50")}
+      size={"small"}
+      variant="bold"
+      style={{ display: "flex", maxWidth: maxWidth }}
```

Comment:
Det blir grymt skumt att lägga maxbredden på texten. Jag tänker att om jag ställer in en maxbredd på min tooltip så är det lådan jag syftar på, inte texten inuti.

Nu blir lådan större än förväntat:
<img width="140" alt="image" src="https://user-images.githubusercontent.com/6987939/214572954-0cd648ad-22b0-4537-915b-ce56fb5c7687.png">